test/common.js

```diff
@@ -265,3 +265,17 @@ export function getBody(win? : SameDomainWindowType = window) : HTMLBodyElement
 
     return win.document.body;
 }
+
+// eslint-disable-next-line no-restricted-globals, promise/no-native
+export async function loadScript(url : string) : Promise<void> {
+    const scriptelement = document.createElement('script');
```

Comment:
Can we camel-case? `scriptElement`

Reply:
Definitively!
Thank you for catching that. It is done now.